tools/run_tests/run_tests.py

```diff
@@ -440,11 +440,15 @@ def dockerfile_dir(self):
   def _get_tox_envs(self, compiler):
     """Returns name of tox environment based on selected compiler."""
     if compiler == 'default':
-      return ('py27', 'py34')
+      return ('py27', 'py34', 'py35', 'py36')
```

Comment:
I'm out of touch with the Dockerfiles, so I can't answer that offhand... but let's start with all of them and see if @jtattermusch tells you to revert any modifications?